addons/website_event/static/src/js/website_event_ticket_details.js

```diff
@@ -35,12 +35,16 @@ odoo.define('website_event.ticket_details', function (require) {
          * @private
          * @param {*} ev
          */
+         /* When the "Fold Tickets Details" option is active, this will be called each
+         time the user expand or fold the tickets (o_wevent_registration_btn). This
+         allows to show/hide elements depending on the folding state. */
```

Comment:
This comments actually goes within the existing comment section (right above the `* @private` line).

This is how comments are written in JavaScript and how we are able to extract them using automated tools.